.pre-commit-config.yaml

```diff
@@ -62,3 +62,8 @@ repos:
   hooks:
     - id: bandit
       args: ["-r", "-ll"]
+
+- repo: https://github.com/twisted/towncrier
+  rev: 23.11.0  # run 'pre-commit autoupdate' to update
+  hooks:
+    - id: towncrier-check
```

Comment:
This ensures that there aren't any fragments in `changes/` with incorrect filenames or change types that don't exist (otherwise it would go unnoticed until an error at release time when running `towncrier build`)